task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -48,7 +51,8 @@ spec:
         mkdir confdir
         if ! oc image extract "${image_with_digest}" --file /bin/opm --file /bin/grpc_health_probe --path $conffolder/*:confdir/ ; then
           echo "Unable to extract image! Skipping checking binaries!"
-          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'Unable to extract image! Skipping checking binaries!')"
+          note="Task $(context.task.name) meets error: Unable to extract image! Skipping checking binaries!"
```

Comment:
SR: note="Task $(context.task.name) Failed: Unable to extract image. Skipping binaries check."

1. Why couldn't it extract the image? What can a user do to try to correct this?
2. Why does this failure also mean that we can't now check binaries?
3. No exclamation points (!) in user help that's this technical. If they're here to indicate importance, we should decide on special formatting to indicate different severities of errors.

Reply:
Good questions, how about change the note to "Task $(context.task.name) failed to extract image by command `oc extract`. Unable to validate the exracted binaries of the image. For details, check the Tekton task log."?